tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+"""Testing configuration file, currently used for generating a Snowpark session for testing."""
+
+import os
+
+import pytest
+from snowflake.snowpark.session import Session
+
+
+@pytest.fixture(scope="module")
+def snowpark_session() -> Session:
```

Comment:
This probably won't work for our testing suite since we dont have snowflake access via actions. Maybe we can use: https://docs.snowflake.com/en/developer-guide/snowpark/python/testing-locally ?